### PR TITLE
Clowns don't get along with rakes

### PR DESCRIPTION
**Pull request by Mickyan**
See here for a demonstration
https://www.youtube.com/watch?v=aRq1Ksh-32g

## Changelog
:cl:
add: Clowns should watch their step around gardening equipment, those rake handles can be unpredictable
/:cl:

### CODE REVIEW

**Comment by carlarctg** [comment]
make it do some brute damage

also make the clowm say deubgrhrhfgh when wearing jester uniform

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 1]
```diff
@@ -75,6 +75,14 @@
 	flags_1 = NONE
 	resistance_flags = FLAMMABLE
 
+/obj/item/cultivator/rake/Crossed(mob/living/carbon/human/H)
+	if(istype(H) && has_gravity(loc) && HAS_TRAIT(H, TRAIT_CLUMSY) && !H.resting)
+		H.confused = max(H.confused, 10)
+		H.Stun(20)
+		playsound(src, 'sound/weapons/punch4.ogg', 50, TRUE)
+		H.visible_message("<span class='warning'>[H] steps on [src] causing the handle to hit [H.p_them()] right in the face!.</span>", \
```
Comment (MrPerson) [inline code/modules/hydroponics/hydroitemdefines.dm]: >right in the face!.

**Comment by kingofkosmos** [comment]
Should there be a random chance of it doing it? Or is it meant to be used as a clown-repellent?

**Comment by OverlyGeneric** [comment]
Needs cartoony noises to accompany with the clown getting hit by them.

Otherwise, 👍

**Comment by Shadowflame909** [comment]
Rakes need to be the glass shards that ash lizards fight against

File: code/modules/hydroponics/hydroitemdefines.dm [hunk 2]
```diff
@@ -75,6 +75,14 @@
 	flags_1 = NONE
 	resistance_flags = FLAMMABLE
 
+/obj/item/cultivator/rake/Crossed(mob/living/carbon/human/H)
+	if(istype(H) && has_gravity(loc) && HAS_TRAIT(H, TRAIT_CLUMSY) && !H.resting)
+		H.confused = max(H.confused, 10)
+		H.Stun(20)
+		playsound(src, 'sound/weapons/punch4.ogg', 50, TRUE)
+		H.visible_message("<span class='warning'>[H] steps on [src] causing the handle to hit [H.p_them()] right in the face!</span>", \
+						  "<span class='userdanger'>You step on the [src] causing the handle to hit you right in the face!</span>")
```
Comment (Urumasi) [inline code/modules/hydroponics/hydroitemdefines.dm]: Won't this produce 'the the rake'?

Comment (PKPenguin321) [reply code/modules/hydroponics/hydroitemdefines.dm]: use \the

**Comment by RaveRadbury** [comment]
Major issue here: Your demonstration involves what is _clearly_ an antag clown, and yet this interaction hinges on the "clumsy" trait _which antag clowns cannot have_

**Comment by nemvar** [comment]
tbh I never got why antag clowns lost clumsy. Like why even play clown if you aren't gonna pull through with it?

**Comment by Mickyan** [comment]
I agree

**Comment by Shadowflame909** [comment]
clown ops still have clumsy

funny watching them lose all their roundstart tools for useless broken stuff like the clown e-sword and try to use the pie cannon only for that to goof up on them

**Comment by Farquaar** [comment]
While potentially quite funny, the cultivator is clearly a hand-tool, not a full blown rake (hence why they can fit in your pocket). I really can't imagine a small tool like that smacking anyone in the face.

Humor has to make sense gosh-diddly-darn it

**Comment by davethwave** [comment]
You can build a rake with 5 wood planks

**Review by 81Denton — APPROVED** [review]

**Comment by ExcessiveUseOfCobblestone** [comment]
@nemvar remove it 😏

**Pull request merged**